ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -70,6 +70,7 @@ public static string ValidYearLaterThanBirthYear(int birthYear)
         public const string TravelOrVisitDurationHasCountry = "Duration cannot be added without a corresponding country";
         public const string VisitCountryRequiresDuration = "Please supply a duration for visit";
         public const string TravelCountryRequiresDuration = "Please supply a duration for travel";
+        public const string SupplyARequiredParam = "You must supply at least one of the required parameters";

```

Comment:
The "Please supply" is a Nancy provided pattern, that I think we should conform to here, though it's not quite the same use-case e.g.

"Please supply at least one search criteria"